wsdl_docs.inc

```diff
@@ -187,12 +187,11 @@ function wsdl_docs_prepare_request_callback($form, $form_state) {
     }
   }
   catch (Exception $e) {
-    //$response = $e->getMessage();
-    drupal_set_message("<pre>Catch Exception - " . $e->getMessage() . "</pre>");
+    drupal_set_message('<pre>' . t('Catch Exception - ') . $e->getMessage() . '</pre>');
```

Comment:
This is catching an exception during the "try it now" operation I think, so displaying a message to the user is appropriate. We don't need to tell them the exception was caught though.

Not sure you need the pre tags, and if you're using t() its better to embed the arguments like `t('Exception -  %msg', array('%msg' => $e->getMessage()));`